Project.toml

```diff
@@ -42,7 +42,7 @@ Bijectors = "0.8"
 Distributions = "0.23.3, 0.24"
 DistributionsAD = "0.6"
 DocStringExtensions = "0.8"
-DynamicPPL = "0.10.2"
+DynamicPPL = "0.10.9"
```

Comment:
I guess we should update the bounds in test/Project.toml as well.

Reply:
Done :+1: